### DISCUSSION TITLE
Cannot stream P2P on HA after upgrade to 2023.10.XX

### DISCUSSION THREAD

**Discussion by AseKarlsson** [fuatakgun/eufy_security#961] [General]
Hi All,

Are there others that cannot start P2P stream from the doorbell (mine T8220-Z) after upgrading Home Assistant to 2023.10.XX and using the example code (this worked fine in 2023.09.XX versions of HA)?

Now on HA 2023.10.03 and eufy 7.7.1 and still not working.

Thanks,
Ase

**Comment by AseKarlsson** [comment]
Some more info:
If I press "Start P2P Stream" in "Settings / Devices & services / Eufy Security / Doorbell" it reacts but the only thing happening is that "Stream Status" goes to "StreamStatus.PREPARING" and "Video Que Size" just keep increasing.

**Comment by fuatakgun** [comment]
Please create an issue and follow the bug template

**Comment by AseKarlsson** [comment]
@fuatakgun, I added the Entry from: 
- Debug Logs from Home Assistant (https://github.com/fuatakgun/eufy_security/discussions/624) - Without this, I will automatically resolve this issue.

But do not get any new info in the Add-On logs after this?

**Comment by AseKarlsson** [comment]
Everything worked fine on HA 2023.09.xx with this same configuration, so something must have change.

**Comment by AseKarlsson** [comment]
Can anyone confirm this working with T8220-Z with P2P on HA 2023.10.xx,  maybe I have to setup everything again?

**Reply by AseKarlsson** [reply]
Sad to see so low usage of this integration :(

**Reply by fuatakgun** [reply]
Why do you think it has low usage?

Use home assistant forums and just check supported devices. You can find answers yourself.

There are more than 2000 installations.

https://analytics.home-assistant.io/custom_integrations.json

**Reply by AseKarlsson** [reply]
Just simply that no one here can confirm this working with HA 2023.10. Worked great with HA 2023.09 but never after that

**Reply by fuatakgun** [reply]
This is not the right place nor are people good at responding at all after solving their own issues.

If it is supported, it has to work.

I have worked remotely and supported them online many users around the world and I personally have it working locally with many different cameras.

And if you think you have encountered a bug, follow the bug template and create a new one.

**Reply by AseKarlsson** [reply]
@fuatakgun, what changed between HA 2023.09 and HA 2023.10 that broke your integration regarding P2P stream? I asked a lot of questions about the logs needed but with now response, 
Finaly some more: https://github.com/fuatakgun/eufy_security/issues/995

**Reply by fuatakgun** [reply]
If you think something has changed between then and now, you can just downgrade the integration and try, right?

And i really don't understand your message about the question part.

**Reply by AseKarlsson** [reply]
It is not your Integration version that is the problem and need to go back. After upgrading HA to version 2023.10 and 2023.11 the P2P stopped working.  Do not want to go back to HA 2023.09

**Reply by AseKarlsson** [reply]
I hope solving #995 will fix this, "known issue" was posted!

**Comment by AseKarlsson** [comment]
I don’t really care about the stream itself, but this was my workaround because I could not attach the “Event Image” into an email, So I started the stream, took an capture, stopped the stream and that could be attach to an email.

**Comment by AseKarlsson** [comment]
P2P streaming started working today after the latest update of WebRTC 3.5.1 that I also saw you have been involved in, Great!

My old trick above to "start stream, take capture, stop stream" wont work fully, I get an email with the capture, but the .jpg cannot be opened, so will continue to try to solve this.

If you find a way to attach the "image.doorbell_event_image" to an email without these steps it would be great!

Thanks,
Ase

**Comment by AseKarlsson** [comment]
Sadly the P2P Streaming stopped working again with HA 2024.1.1 and 2024.1.2

**Comment by AseKarlsson** [comment]
P2P works again with  HA OS 11.4, but not the Snapshot for the email.
